### ISSUE TITLE
Console warning on subsequent startups after setting { index: false }

### ISSUE THREAD

**Issue by kjlape** [Meteor-Community-Packages/meteor-collection2#118]
# Question

On line 88 in collection2.js at the latest version (311c35e8), there is a console.warn that occurs when trying to drop an index which doesn't exist.  I understand that you don't want to just swallow the exception, but I'm also wondering if there is a better solution.  I'm happy to submit a pull request, but I'm not familiar enough with this package to come to a well reasoned solution.  A `silentIndexDrop` flag perhaps?  Does anyone have an idea for this? @aldeed?
# Background

If I'm developing a package/project which will be used in unforeseen number of situations and decide to drop an index using the { index: false } method, I have to leave it in the schema in the interest of making sure that index gets dropped on all deployments.  I can live with the resulting warnings in the log, but it is really just extra noise to me.
# TL;DR

I don't want a warning here. Can we fix it?
